src/components/Login/LoginStyle.tsx

```diff
@@ -0,0 +1,24 @@
+import styled, { css } from 'styled-components'
```

Comment:
remove components folder and adding styles to Login component and moving component to the Login component